app/services/doc_auth/socure/responses/docv_result_response.rb

```diff
@@ -100,7 +100,9 @@ def successful_result?
         end
 
         def error_messages
-          if !successful_result?
+          if !id_type_supported?
+            { unaccepted_id_type: 'failed' }
```

Comment:
For network errors with no additional information available, we just say `network: true`. Do we want to follow the same pattern here, and say `unaccepted_id_type: true`? (If we have more information, of course, we could say something like `{unaccepted_id_type: 'library_card'}`, but I don't think we do here.)